web_last_viewed_records/static/src/xml/main.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<templates>
+    <t t-name="web_last_viewed_records.header">
+        <tr class="oe_header_row">
+            <td t-att-colspan="colspan or '3'">
+                <div class="oe_view_manager_last_viewed"/>
+            </td>
+            <td></td>
```

Comment:
As I remember, I added extra **td** to put there some control buttons in the feature.